about.html

```diff
@@ -0,0 +1,142 @@
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/58789452ae.js"
+      crossorigin="anonymous">
+    </script>
+    <link rel="stylesheet" href="index.css">
+    <title>EDULEARN CONFERENCE 2023</title>
+</head>
+
+<body>
+<header>
+  <div class="top-nav"> 
+    <a href="#" target="_blank"><i class="fa-brands fa-facebook-f"></i></a>
+    <a href="#" target="_blank"><i class="fa-brands fa-twitter"></i></a>
+    <a href="#">English</a>
+    <a href="#">My Page</a>
+    <a href="#">LogOut</a>
+    
+    </div>
+<div class="menubar">
+   <div class="navbar">
+    <nav>
+        <img src="images/Union.png" alt="menu">
+    </nav>
+    </div>
+    <nav class="navbar1">
+      <img src="images/logo.png"  alt="logo for convocation"> 
+        <ul>
+          <a  href="index.html">Home</a>
+          <a  href="index.html#courses">Courses</a>
+          <a  href="index.html#speakers">Speakers</a>
+          <a  href="index.html#partners">Sponsers</a>
+          <a  href="index.html">News</a>  
+        </ul>
+    </nav>
+</div>
+</header>
+
+<div class="menuclick" id="nav">
+    <div class="exit">
+      <button>
+        &#x2716;
+      </button>
+    </div>  
+             <div class="menu_details">
+              <a  href="index.html">Home</a>
+              <a  href="index.html#courses">Courses</a>
+              <a  href="index.html#speakers">Speakers</a>
+              <a  href="index.html#partners">Sponsers</a> 
+              <a  href="index.html">News</a>  
+              
+             </div>
+ </div>
+        <section class="main-pg">
+                <div class="main_box">
+                    <h1 class="main_heading">"Hello! Educating World"</h1>
+                    <h2 class="main_title">EDULEARN CONFERENCE 2023</h2>
+                    <hr id="hr-hero">
+                </div>
+                <div class="main_descriptionA">
+                    <p class="des1A">About Conference: Welcome to the official website International Conference on Education and New Learning Technologies (EDULEARN). The conference provides a platform for professionals involved in Education and New Learning Technologies to exchange knowledge and gain an insight into the state of the art in the current technology, techniques and solutions in Education and New Learning Technologies as they have been developed and applied in different countries. Participants include a wide variety of stakeholders from research and academia, to industrial sectors as well as government organizations.</p>
+                </div>
+            <div class="contact">
+                <p class="reach_out">Please contact us at :-  contact@worldresearchsociety.com</p>
+                <span>
+                    <a href=" mailto:contact@worldresearchsociety.com" class="email">Click here:</a>
+                    <img src="images/arrow.png" id="arrow" alt="arrow">
+                </span>
```

Comment:
- [ ] For desktop size On the About page from the design It contains a top section with a centered column that includes a page title, a text block with the page description, and a line with a contact information or any other prompt under it, kindly adjust the alignment of the  page title, a text block with the page description, and a line with a contact information just like the design guidelines.
![Capture](https://user-images.githubusercontent.com/93598189/219815283-3ed817c0-c3e2-41d1-a73d-b485683ced93.PNG)

Your Project Design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/219815498-b77efa51-6cb4-49b9-b6fc-99e5ac6684d7.PNG)